src/test/java/seedu/internbuddy/logic/commands/CommandTestUtil.java

```diff
@@ -57,7 +57,6 @@ public class CommandTestUtil {
     public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "invalid!email"; // missing '@' symbol
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
-    public static final String INVALID_STATUS_DESC = " " + PREFIX_STATUS;
```

Comment:
What does this do

Reply:
It wasn't used, so I removed it from the code!